### ISSUE TITLE
[v1.0.0b] - Recipe Deletion Bugs

### ISSUE THREAD

**Issue by coffee-mage** [mealie-recipes/mealie#770]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

1) If you delete recipe on All Recipes screen.   It gets deleted in the backend, but recipe doesn't disappear until you reload the page.  If you click the non-existent recipe, it loads blank screen.
2) If you delete a recipe (e.g. Baked Meatballs), and then make a new recipe with the exact same name.  The newly created recipe has the image from the old recipe.  I'm not sure if recipe delete is not deleting the assets properly, or if this is a feature.  :)

Let me know if any of these don't make sense.  Happy to give more detailed information.

### Deployment

Other

### Deployment Details

Linux,  git-hub clone, and development scripts.  SQLite database.

**Comment by hay-kot** [comment]
Was able to get these fixed in 

- #778

**Comment by coffee-mage** [comment]
Delete seems to work in the interface now.  But when you delete, it feels like the image file is not getting deleted.

As an example, if I delete a recipe (with picture) called Pizza.   Then create a new recipe called Pizza,  that new recipe is autopopulated with the picture from the deleted recipe.

**Comment by hay-kot** [comment]
I did verify it gets deleted. The problem is, thay the image gets cached in your browser, so when you create a recipe with the same name it's using the cached image instead of fetching a new one.

AFAIK there is no API for clearing the browser cache for a url from Javascript, so I left thay one alone. Figured it was an edge case. One possibility would be to generate unique id's in the backend for the image URLs, but that may harm performance overall.
